pkg/apiserver/metrics/metrics.go

```diff
@@ -48,6 +48,8 @@ var (
 		prometheus.SummaryOpts{
 			Name: "apiserver_request_latencies_summary",
 			Help: "Response latency summary in microseconds for each verb and resource.",
+			// Make the sliding window of 1h.
+			MaxAge: time.Hour,
```

Comment:
Does this need to be the same as MaxAge in client/metrics/metrics.go?  I assume so, in which case you need a shared constant, to prevent them from getting out of sync is someone changes one and not the other.